src/analysis/raster/qgsninecellfilter.h

```diff
@@ -37,16 +37,29 @@ class QgsFeedback;
 class ANALYSIS_EXPORT QgsNineCellFilter
 {
   public:
+    //! Result of the calculation \since QGIS 3.44
+    enum Result
+    {
```

Comment:
Can this be an enum class?

Reply:
We'd also need to verify that int/bool comparison stills from python for API stability

Reply:
Should I also update other similar enums to enum classes? E.g. in the kernel density estimation and raster calculator.

Reply:
It's desirable, but not mandatory for YOU to do it :stuck_out_tongue_closed_eyes:

Reply:
Ok, I will do it in a separate pull-request.